src/server/sams/assets/service.py

```diff
@@ -56,6 +57,26 @@ def post(self, docs: List[Dict[str, Any]], **kwargs) -> List[ObjectId]:
             file_meta = self.upload_binary(doc, content)
             doc.update(file_meta)
 
+            # Add original rendition to the asset renditions
```

Comment:
This code block would need to be wrapped in a `try...except` clause, so that this code block only runs if the Asset is an image